### PR TITLE
docker: initial commit of dockerfile

### PR DESCRIPTION
**Pull request by Sirupsen**
This'll make it even easier to include it for development and CI in other projects that already use Docker and is extremely simple to do with Go.

@xthexder

### CODE REVIEW

File: toxiproxy.go [hunk 1]
```diff
@@ -12,7 +12,7 @@ var host string
 var port string
 
 func init() {
-	flag.StringVar(&host, "host", "localhost", "Host for toxiproxy's API to listen on")
+	flag.StringVar(&host, "host", "0.0.0.0", "Host for toxiproxy's API to listen on")
```
Comment (Sirupsen) [inline toxiproxy.go]: This is so it binds to all interfaces in Docker which makes sure that if you're using native networking it binds on the `docker0` interface so you can access it from other containers.

Comment (xthexder) [inline toxiproxy.go]: This kindof seems wrong to me. Depending on where you're running it, you don't want just anyone to go and mess with your proxies.

Is there some way to just pass `-host 0.0.0.0` in the Dockerfile?

Comment (Sirupsen) [inline toxiproxy.go]: Yeah, we can just do that in `CMD`.

**Comment by xthexder** [comment]
:+1: :cake:

**Pull request merged**